.github/workflows/docs.yaml

```diff
@@ -3,10 +3,10 @@ name: Docs
 on:
   push:
     branches: [master]
-    paths-ignore: ['**/*.md']
+    paths-ignore: ['*.md', '**/*.md']
```

Comment:
Reading [the docs](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet), I think we want `**.md` in place of both of these.

Reply:
Updated. Hopefully that will work... the docs seem to say that `**/*.md` should have caught all markdown files in the repo. :/